sql/core/src/test/resources/sql-tests/results/explain.sql.out

```diff
@@ -0,0 +1,714 @@
+     
+(15) Project [codegen id : 1]
+Output    : [key#x]
+Input     : [key#x, val#x]
+     
+(16) HashAggregate [codegen id : 1]
+Input: [key#x]
+     
+(17) Exchange 
+Input: [max#x]
+     
+(18) HashAggregate [codegen id : 2]
+Input: [max#x]
+
+
+-- !query 10
+EXPLAIN FORMATTED
+  SELECT * 
+  FROM   explain_temp1 
+  WHERE  key = (SELECT max(key) 
+                FROM   explain_temp2 
+                WHERE  val > 0) 
+         OR
+         key = (SELECT max(key) 
+                FROM   explain_temp3
+                WHERE  val > 0)
+-- !query 10 schema
+struct<plan:string>
+-- !query 10 output
+== Physical Plan ==
+* Filter (3)
++- * ColumnarToRow (2)
+   +- Scan parquet default.explain_temp1 (1)
+
+
+(1) Scan parquet default.explain_temp1 
+Output: [key#x, val#x]
+     
+(2) ColumnarToRow [codegen id : 1]
+Input: [key#x, val#x]
+     
+(3) Filter [codegen id : 1]
+Input     : [key#x, val#x]
+Condition : ((key#x = Subquery scalar-subquery#x) OR (key#x = Subquery scalar-subquery#x))
+     
+===== Subqueries =====
+
+Subquery:1 Hosting operator id = 3 Hosting Expression = Subquery scalar-subquery#x
+* HashAggregate (10)
++- Exchange (9)
+   +- * HashAggregate (8)
+      +- * Project (7)
+         +- * Filter (6)
+            +- * ColumnarToRow (5)
+               +- Scan parquet default.explain_temp2 (4)
+
+
+(4) Scan parquet default.explain_temp2 
+Output: [key#x, val#x]
+     
+(5) ColumnarToRow [codegen id : 1]
+Input: [key#x, val#x]
+     
+(6) Filter [codegen id : 1]
+Input     : [key#x, val#x]
+Condition : (isnotnull(val#x) AND (val#x > 0))
+     
+(7) Project [codegen id : 1]
+Output    : [key#x]
+Input     : [key#x, val#x]
+     
+(8) HashAggregate [codegen id : 1]
+Input: [key#x]
+     
+(9) Exchange 
+Input: [max#x]
+     
+(10) HashAggregate [codegen id : 2]
+Input: [max#x]
+     
+Subquery:2 Hosting operator id = 3 Hosting Expression = Subquery scalar-subquery#x
+* HashAggregate (17)
++- Exchange (16)
+   +- * HashAggregate (15)
+      +- * Project (14)
+         +- * Filter (13)
+            +- * ColumnarToRow (12)
+               +- Scan parquet default.explain_temp3 (11)
+
+
+(11) Scan parquet default.explain_temp3 
+Output: [key#x, val#x]
+     
+(12) ColumnarToRow [codegen id : 1]
+Input: [key#x, val#x]
+     
+(13) Filter [codegen id : 1]
+Input     : [key#x, val#x]
+Condition : (isnotnull(val#x) AND (val#x > 0))
+     
+(14) Project [codegen id : 1]
+Output    : [key#x]
+Input     : [key#x, val#x]
+     
+(15) HashAggregate [codegen id : 1]
+Input: [key#x]
+     
+(16) Exchange 
+Input: [max#x]
+     
+(17) HashAggregate [codegen id : 2]
+Input: [max#x]
+
+
+-- !query 11
+EXPLAIN FORMATTED
+  SELECT (SELECT Avg(key) FROM explain_temp1) + (SELECT Avg(key) FROM explain_temp1)
+  FROM explain_temp1
+-- !query 11 schema
+struct<plan:string>
+-- !query 11 output
+== Physical Plan ==
+* Project (3)
++- * ColumnarToRow (2)
+   +- Scan parquet default.explain_temp1 (1)
+
+
+(1) Scan parquet default.explain_temp1 
+Output: []
```

Comment:
what happened here？Why the output is Nil?

Reply:
@cloud-fan Because of column pruning ..